src/clj/rems/db/test_data.clj

```diff
@@ -962,6 +962,41 @@
         form (create-all-field-types-example-form! owner {:organization/id "nbn"} "Example form with all field types" {:en "Example form with all field types"
                                                                                                                        :fi "Esimerkkilomake kaikin kenttätyypein"
                                                                                                                        :sv "Exempelblankett med alla fälttyper"})
+
+        form-with-public-and-private-fields (create-form! {:actor owner
+                                                           :organization {:organization/id "nbn"}
+                                                           :form/internal-name "Public and private fields form"
+                                                           :form/external-title {:en "Form"
+                                                                                 :fi "Lomake"
+                                                                                 :sv "Blankett"}
+                                                           :form/fields [{:field/title {:en "Simple text field"
+                                                                                        :fi "Yksinkertainen tekstikenttä"
+                                                                                        :sv "Textfält"}
+                                                                          :field/optional false
+                                                                          :field/type :text
+                                                                          :field/max-length 100}
+                                                                         {:field/title {:en "Private text field"
+                                                                                        :fi "Yksityinen tekstikenttä"
+                                                                                        :sv "Privat textfält"}
+                                                                          :field/optional false
+                                                                          :field/type :text
+                                                                          :field/max-length 100
+                                                                          :field/privacy :private}]})
+
+        form-private-nbn (create-form! {:actor owner
+                                        :organization {:organization/id "nbn"}
+                                        :form/internal-name "Simple form"
```

Comment:
Since we have multiple forms with the same name, we can check the UX for this and later improve if needed (I think we see the org everywhere it matters so one can distinguish these).